### ISSUE TITLE
Remove company name from Apache License Files

### ISSUE THREAD

**Issue by fdegir** [eiffel-community/community#19]

**Comment by henning-roos** [comment]
The Appendix includes a boilerplate header info for source files in respective repository. This boilerplate should not contain a reference to a specific company, but specific source files can (should?) contain name of copyright owner. If I have understood the license correctly.

**Comment by henning-roos** [comment]
The following repositories have faulty license file:
https://github.com/eiffel-community/eiffel
https://github.com/eiffel-community/community/
https://github.com/eiffel-community/eiffel-intelligence/
https://github.com/eiffel-community/eiffel-commons/
https://github.com/eiffel-community/eiffel-community.github.io/
https://github.com/eiffel-community/eiffel-gerrit-herald/
https://github.com/eiffel-community/eiffel-gerrit-lib/
https://github.com/eiffel-community/eiffel-vici/


The following repositories does not have any license file at all:
https://github.com/eiffel-community/eiffel-store
https://github.com/eiffel-community/eiffel-remrem-protocol-interface


I will create PR:s for each of these repositories in order to remedy this situation.

**Comment by henning-roos** [comment]
Related but not part of this issue:
https://github.com/eiffel-community/eiffel-remrem-protocol-interface/issues/27
https://github.com/eiffel-community/eiffel-store/issues/6

**Comment by henning-roos** [comment]
Waiting on @eiffel-community/eiffel-protocol-maintainers to review https://github.com/eiffel-community/eiffel/pull/220 before this issue can be closed.

**Comment by henning-roos** [comment]
All PR:s merged. Closing this issue...
